src/home/home.tsx

```diff
@@ -26,6 +33,41 @@ const Home = () => {
     revalidateOnReconnect: false,
   })
 
+  const {data: configResponse} = useSWR(configUrl, fetcher, {
+    revalidateIfStale: false,
```

Comment:
can we have a variable for SWR options & use it all the places?

Reply:
changes have been made, extracted to a common place